doc.txt

```diff
@@ -1,3 +1,8 @@
+/**
+* @defgroup configurations Compile time configurations
+* @brief List of compile-time configurations
+*/
```

Comment:
Add some description of what is supposed to go there, so that people don't have to go to PR comments to find out.

Reply:
Other than that I'm fine, and I would approve this stuff.

Reply:
done. How does it look now?